### DISCUSSION TITLE
Update README to include information on how to specify a version of a lib to install

### DISCUSSION THREAD

**Discussion by netpoetica** [microsoft/vcpkg#22270] [Q&A]
[Issue 1681](https://github.com/microsoft/vcpkg/issues/1681) was [closed with a note ](https://github.com/microsoft/vcpkg/issues/1681#issuecomment-967338838)from @ras0219-msft that the feature was implemented. A [document ](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md)was added that instructs package publishers how to specify versions.

Neither that document, or the root level README, has any *user-facing instructions* on how to do something like:

`vcpkg install mypackage@v1.0.0`

Documentation should be provided not only for package authors, but more importantly for package consumers.

**Comment by dg0yt** [comment]
The mentioned note does point to user-facing instructions. However, it implies that you consume packages via manifest mode. Your command line is "classic mode".

(BTW package authors cannot express version requirements. Every single git revision of the vcpkg repo is meant to form a consistent universe.)

**Comment by ras0219-msft** [comment]
>package authors cannot express version requirements.

Package authors can express `>=` constraints on other packages (modeling "I need at least the version when API X was added"): https://github.com/microsoft/vcpkg/blob/master/docs/users/manifests.md#version-field.

If your package consumption scenario requires finer control over versions, we strongly recommend you move to Manifest mode. Simply run `vcpkg install` in a directory containing an appropriate `vcpkg.json` (see https://github.com/microsoft/vcpkg/blob/master/docs/users/manifests.md for details) and you have 100% control over the versions of dependencies used including pinning back older versions and pulling from private registries.

**Comment by dg0yt** [comment]
> Package authors can express `>=` constraints on other packages

Thanks for clarifying. Probably it is not very well known that this aspect of *user* documentation also applies for *maintainers*: only 9 ports use such expressions. (Even if it is not so important within vcpkg, it is relevant for manifest mode users.)
